public/cli/lib/rendering/renderFrames.spec.ts

```diff
@@ -0,0 +1,60 @@
+import test from 'ava';
+import { getHtml } from './renderFrames';
+import { AnimationSettings, Timings } from '../../../models';
+
+test('renderFrames smoke test', async (t) => {
+  // TODO implement this
+  t.pass();
+});
+
+test('getHtml() loads html from template', async (t) => {
+  let style = mockStyle();
+  let timings = mockTimings();
+  let htmlContent = await getHtml(timings, style);
+  let regexPatterns = [
```

Comment:
If the value doesn't change use `const`.

Reply:
done.